var/spack/repos/builtin/packages/py-openai-gym/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyOpenaiGym(PythonPackage):
+    """OpenAI Gym is a toolkit for developing and comparing
+    reinforcement learning algorithms. This is the gym open-source
+    library, which gives you access to a standardized set of
+    environments."""
+
+    homepage = "https://github.com/openai/gym"
+    url      = "https://github.com/openai/gym/archive/refs/tags/0.18.1.tar.gz"
+
+    version('0.18.1', sha256='dc59d705e9668c68835fa65cabd7947aac694cefc7843669566e500db4bdaeab')
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-scipy', type=('build', 'run'))
+    depends_on('py-numpy@1.10.4:', type=('build', 'run'))
+    depends_on('py-pyglet@1.4.0:1.5.15', type=('build', 'run'))
+    depends_on('py-pillow@:8.2.0', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('pil@:8.2.0', type=('build', 'run'))
```
We have a virtual package for this